index.js

```diff
@@ -54,6 +54,13 @@ const reportMissingRequiredFlags = missingRequiredFlags => {
 	}
 };
 
+const reportUnknownFlags = unknownFlags => {
+	console.error(`Unknown flag${unknownFlags.length > 1 ? 's' : ''}`);
+	for (const flag of unknownFlags) {
+		console.error(flag);
+	}
```

Comment:
It's better to only call `console.error` once for the whole output.

```js
unknownFlags.join('\n');
```

Reply:
Thanks, all fixed ☕️